lib/ios.js

```diff
@@ -226,9 +226,27 @@ class OptionalIOSWebkitDebugProxyCommandCheck extends DoctorCheck {
 }
 checks.push(new OptionalIOSWebkitDebugProxyCommandCheck());
 
+class OptionalIfuseCommandCheck extends DoctorCheck {
+  async diagnose () {
+    const ifusePath = await resolveExecutablePath('ifuse');
+    if (ifusePath) {
+      const {stdout, stderr} = (await exec(ifusePath, ['-V']));
+      const version = `${stdout.trim()}${stderr.trim()}`;
```

Comment:
does it write into both streams?

Reply:
On my env, the message appeared in `stderr`, not `stdout`.
Not sure other environment cases such as Windows, so, I put both outputs in case of `stdout`

Reply:
it should be always stderr then